### PR TITLE
fix #9813 feat(nimbus): Disable fields for the web experiments

### LINKED ISSUE #9813: Hide fields which are not applicable to Monitor
Some fields are not applicable to the Monitor like Firefox 

```
min version- Audience page
max version- Audience page
countries- Audience page
Locales- Audience page
localization check box - Is this a localized experiment?- Branches page
Sticky Enrollment (Clients remain enrolled until the experiment ends)- Audience page
first run, first run release date- Audience page
```

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-3746)

### PR DESCRIPTION
**Pull request by yashikakhurana**
Because

- Some fields on the Audience page are not applicable for Web based applications like Monitor, VPN etc

This commit

- Disable the following fields and added the tool tip on top of each fields when it is disabled
```
min version- Audience page
max version- Audience page
countries- Audience page
Languages- Audience page
Sticky Enrollment (Clients remain enrolled until the experiment ends)- Audience page
```
Screenshot 2023-11-24 at 10 09 41 AM

fixes #9813

### CODE REVIEW

File: experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.test.tsx [hunk 1]
```diff
@@ -2115,6 +2115,42 @@ it("enrollment period is not diabled when experiment is not a rollout", async ()
...
+  const selectCountriesElement = countriesDiv?.querySelector("input");
+  expect(selectCountriesElement).toHaveAttribute("disabled");
+  const languagesDiv = document.querySelector('[data-testid="languages"]');
+  const selectLanguagesElement = languagesDiv?.querySelector("input");
+  expect(selectLanguagesElement).toHaveAttribute("disabled");
+  expect(screen.queryByTestId("isSticky")).toBeDisabled();
+  screen.getByTestId("tooltip-disabled-is-sticky");
+  expect(screen.getByTestId("isFirstRun")).toBeDisabled();
+  screen.getByTestId("tooltip-disabled-first-run");
+  expect(screen.queryByTestId("proposedReleaseDate")).toBeDisabled();
```
Comment (eliserichards) [inline experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.test.tsx:2152]: Nit: can we add some whitespace in here to make it a little more readable? Maybe group all of the `screen.getByTestId`s together and the `expect`s together, or but a newline in between each pair?

File: experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx [hunk 2]
```diff
@@ -472,11 +473,23 @@ export const FormAudience = ({
           <Form.Group as={Col} controlId="minVersion">
             <Form.Label className="d-flex align-items-center">
               Min Version
+              {experiment.isWeb && (
+                <>
+                  <Info
+                    data-tip={TOOLTIP_DISABLED_FOR_WEBAPP}
```
Comment (eliserichards) [inline experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx:479]: 🎉🎉🎉

File: experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx [hunk 3]
```diff
@@ -472,11 +473,23 @@ export const FormAudience = ({
...
+                  />
+                  <ReactTooltip />
+                </>
+              )}
             </Form.Label>
             <Form.Control
               {...formControlAttrs("firefoxMinVersion")}
               as="select"
-              disabled={isLocked!}
+              disabled={isLocked! || experiment.isWeb}
```
Comment (eliserichards) [inline experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx:492]: could we add this to the `isLocked` definition? 

```
const isLocked = (isLiveRollout && !getStatus(experiment).draft) || experiment.isWeb;
```

Comment (eliserichards) [reply experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx:492]: or are there fields that we want locked using `isLocked` that don't line up with locked for web?

**Review by eliserichards — APPROVED** [review]
Looking good! 💯  Just a few nits:

File: experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx (hunk 3 above)
Comment (yashikakhurana) [reply experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx:492]: yes there are certain fields which are not consistent with `isLocked`

Comment (yashikakhurana) [reply experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx:492]: :(

File: experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.test.tsx (hunk 1 above)
Comment (yashikakhurana) [reply experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.test.tsx]: done done done 🕺

**Pull request merged**